src/System.Memory/src/System/Span.cs

```diff
@@ -233,6 +233,109 @@ internal Span(Pinnable<T> pinnable, IntPtr byteOffset, int length)
         }
 
         /// <summary>
+        /// Clears the contents of this span.
+        /// </summary>
+        public unsafe void Clear()
+        {
+            var length = _length;
```

Comment:
WTH! My bad really thought I went through everything, but apparently missed the most obvious one.